Lib/test/test_dataclasses.py

```diff
@@ -68,6 +68,17 @@ def test_field_repr(self):
 
         self.assertEqual(repr_output, expected_output)
 
+    def test_dataclass_params_repr(self):
```

Comment:
Is it worthwhile to do a test based on `for param in inspect.signature(dataclass).parameters` check? So new flags will automatically fail this test?

Reply:
Added!